app/workers/rapid_ready_for_decision/form526_hypertension_job.rb

```diff
@@ -22,7 +22,7 @@ def query_and_assess_lighthouse(form526_submission)
       client = lighthouse_client(form526_submission)
       bp_readings = assess_bp_readings(client.list_resource('observations'))
       # stop querying if bp_readings.blank?
-      medications = assess_medications(client.list_resource('medications')) if bp_readings.present?
+      medications = assess_medications(client.list_resource('medication_requests')) if bp_readings.present?
```

Comment:
in the future do we want to refer to these as `medicationRequests` in variable naming as well? lots of places where this nuance is lost (confusingly, `HypertensionMedicationRequestData` seems to be named accidentally-correctly but refers to it only as `Medication` in a few spots...)